strawberry/django/__init__.py

```diff
@@ -0,0 +1,6 @@
+try:
+    # import modules and objects from external strawberry-graphql-django
+    # package so that it can be used through strawberry.django namespace
+    from strawberry_django import *  # noqa: F401, F403
+except ModuleNotFoundError:
+    pass
```

Comment:
Do we want to add a custom [module `__getattr__`](https://www.python.org/dev/peps/pep-0562/) that tells the user to install the extension package if they try using it without installing it?

Reply:
I thought about it and I'd like to do that.

The challenge is that we have few modules already under django subfolder and we should probably move them into somewhere else if would like to do that.

Let's take an example.
```python
# this should raise an exception and nice tip
strawberry.django.type

# this and few other submodules should still work
strawberry.django.views.GraphQLView
```

Reply:
https://github.com/strawberry-graphql/strawberry/issues/849 should hopefully help with that

Reply:
> Do we want to add a custom module __getattr__ that tells the user to install the extension package if they try using it without installing it?

I tried module level `__getattr__` and it seems to work pretty well. 

Please let me know if this is feasible solution or if you think I need to change anything else.

Reply:
Can you still use `strawberry.django.views.GraphQLView` if the extension isn't installed?

Reply:
Yes, you can use GraphQLView in the way we have it in our documentation.
https://strawberry.rocks/docs/integrations/django

```python
from strawberry.django.views import GraphQLView
```

but unfortunately following does not work anymore because it tries to access `views` through `__getattr__`
```python
from strawberry.django import views
```

do you think if that's a problem? I can remove the last commit if you think so.

Reply:
Can we just add a bit of logic to `__getattr__` to try the import, and if that fails only then throw an exception?

Reply:
Probably yes, Let me try :)

Reply:
https://stackoverflow.com/a/49412969/8134178

Oh this may not even be a problem

Reply:
All following import variants are working with latest change
```python
from strawberry.django import views
from strawberry.django.views import GraphQLView
import strawberry.django.views
```

Reply:
I pushed suggested changes to the branch. Test are passing now after changing exception from `ModuleNotFoundError`to `AttributeError`.

Reply:
Haha, README.md check did not pass. Need to fix that too :)

Reply:
pre-commit check still fails because of an errant trailing space :slightly_smiling_face:

Reply:
> pre-commit check still fails because of an errant trailing space 🙂

fixed :)